fe1-web/src/features/lao/network/LaoGreetWatcher.ts

```diff
@@ -48,16 +48,24 @@ export const storeBackendAndConnectToPeers = (
   // address, otherwise a client will connect to the same server twice (e.g. using its IP and then
   // then using the canonical domain address)
   const networkManager = getNetworkManager();
-  for (const peerAddress of greetLaoMsg.peers) {
-    networkManager.connect(peerAddress.address);
-  }
 
-  // mark the lao#greet message as handled
-  dispatch(
-    handleGreetLaoMessage({
-      messageId: messageId.valueOf(),
-    }),
-  );
+  try {
+    await Promise.all(
+      greetLaoMsg.peers.map((peerAddress) => networkManager.connect(peerAddress.address)),
+    );
+
+    // mark the lao#greet message as handled
+    dispatch(
+      handleGreetLaoMessage({
+        messageId: messageId.valueOf(),
+      }),
+    );
+  } catch {
+    // here we can actually ignore errors for now since
```

Comment:
since this is a "for now" thing, I'd suggest adding a console.warn indicating that this is currently an unsupported / unimplemented path. This way it's easier to find than a subtly hidden unimplemented feature 😉